pkgs/applications/video/mythtv/patches/fix-os-detection.patch

```diff
@@ -0,0 +1,12 @@
+--- a/configure	1970-01-01 01:00:01.000000000 +0100
++++ b/configure	2019-10-25 19:29:19.948540422 +0200
+@@ -6645,8 +6645,7 @@
+         case $target_os in
+             linux)
+                 # Bah. Suse linux doesn't have xnvctrl.
+-                . /etc/os-release
+-                case $ID in
++                case nix in
```

Comment:
not sure how i feel about this one

Reply:
I've made another fix which might be a better way to patch it.

Reply:
I think you can remove this file now?